src/main/java/io/confluent/connect/jdbc/source/JdbcSourceTaskConfig.java

```diff
@@ -29,9 +29,12 @@ public class JdbcSourceTaskConfig extends JdbcSourceConnectorConfig {
 
   public static final String TABLES_CONFIG = "tables";
   private static final String TABLES_DOC = "List of tables for this task to watch for changes.";
-
+  public static final String TABLES_FETCHED = "tables.fetched";
```

Comment:
hmm not sure this is a user level config? What values are they going to provide in this? If the purpose of this config is to check if the tables have been fetched or not, it can be done by a variable as well. Not sure I follow this..

Reply:
This is not user level config. This is internal config of the task itself. Can't use variable for this status of query is fetched from Connector class whereas this has to be used in the task class.

Reply:
Ok. Having it exposed via config is  still not the right way to go. You can look at the interface `SourceTaskContext` which is exposed by the framework. I think it's designed for such cases. Also, IIRC we had also discussed a case where the query failed and the. You can even haver an AtomicBoolean object instantiated in connector and updated by the Task or TableMonitoringThread.

Reply:
Resolved offline. We can't use SourceTaskContext here and the config we have exposed is internal config and not user config. This pattern is already being used in this Connector with the TABLES config in the task.

Reply:
please add details in the PR description on why SourceTaskContext cannot be used.